src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -79,11 +82,15 @@ enum DruidSpells
     SPELL_DRUID_INNERVATE_RANK_2               = 326228,
     SPELL_DRUID_INFUSION                       = 37238,
     SPELL_DRUID_LANGUISH                       = 71023,
+    SPELL_DRUID_LIFEBLOOM                      = 33763,
+    SPELL_DRUID_LIFEBLOOM_OVERGROWTH           = 188550,
     SPELL_DRUID_LIFEBLOOM_FINAL_HEAL           = 33778,
     SPELL_DRUID_LUNAR_INSPIRATION_OVERRIDE     = 155627,
     SPELL_DRUID_MANGLE                         = 33917,
     SPELL_DRUID_MASS_ENTANGLEMENT              = 102359,
     SPELL_DRUID_MOONFIRE_DAMAGE                = 164812,
+    SPELL_DRUID_PHOTOSYNTHESIS                 = 274902,
+    SPELL_DRUID_PHOTOSYNTHESIS_EFFECT          = 274906,
```

Comment:
I think either the comment below is wrong or you mixed the ids here?

Reply:
no, it's correct